enclone_core/src/combine_group_pics.rs

```diff
@@ -22,8 +22,8 @@ pub fn combine_group_pics(
     if noprint && parseable_stdouth && !group_pics.is_empty() {
         let mut rows = Vec::<Vec<String>>::new();
         for pic in group_pics {
-            let r = pic.split('\n');
-            for rj in r.skip(1) {
+            let r: Vec<_> = pic.split('\n').collect();
+            for rj in r.iter().take(r.len() - 1) {
```

Comment:
Why is this `collect` necessary?  I don't think it should be necessary.  If the objective is simply to omit the last element, then just call [next_back](https://doc.rust-lang.org/stable/std/iter/trait.DoubleEndedIterator.html#tymethod.next_back) on it.

Reply:
https://play.rust-lang.org/?version=nightly&mode=debug&edition=2021&gist=21e343245c346afa7ac456ff08c3cdfa

Reply:
```suggestion
            let r = pic.split('\n');
            r.next_back();
            for rj in r {
```

Reply:
Nice, TIL about `next_back`. I just reverted the code to its previous state. I don't think this function comes anywhere near the critical path, so getting rid of the allocation doesn't save us much.

Reply:
My issue with leaving sub-par code lying around, even if it's not on the critical path, is that those patterns can then end up replicated into the critical path  by copy/pasta or by simple mimicry.

Reply:
Done!